pkg/config/validators.go

```diff
@@ -9,6 +9,11 @@ var (
 	ErrAdminAPIKeyNotConfigured = errors.New("you have not configured your admin API key yet")
 	// ErrApplicationIDNotConfigured is the error returned when the loaded profile is missing the application_id property
 	ErrApplicationIDNotConfigured = errors.New("you have not configured your Application ID yet")
+
+	// ErrCrawlerAPIKeyNotConfigured is the error returned when the loaded profile is missing the crawler_api_key property
+	ErrCrawlerAPIKeyNotConfigured = errors.New("you have not configured your Crawler API key yet")
```

Comment:
Is it enough to show only the error to the user? 
Should we explain how to configure this or add a link to a documentation page?

Reply:
Good point 👍 

I will make those more explicit.

Reply:
Added a `PersistentPreRunE` and a small paragraph to the `Long` attribute of the top level `crawler` command. If no credentials are found, the following message will be displayed:

```
In order to use the 'crawler' commands, you will need to authenticate with the Algolia Crawler API. You can do so by either:
  - Export your Algolia Crawler username and API Key as ALGOLIA_CRAWLER_USER_ID and ALGOLIA_CRAWLER_API_KEY environment variables.
  - Add your Algolia Crawler 'crawler_user_id' and 'crawler_api_key' credentials to your profile file (~/.config/algolia/config.tml).`
  ```